File: loran_js/Tower.js
const DMS = require("geographiclib-dms");

class Tower {
  /**
   * @param {String} lat
   * @param {String} lon
   * @param {Number} codingDelay
   */
  constructor(lat, lon, codingDelay = 0) {
    this.codingDelay = codingDelay;
    
    lat = lat.replace("° ", ":");
    lat = lat.replace("' ", ":");
    lat = lat.replace("\"", "");
    
    lon = lon.replace("° ", ":");
    lon = lon.replace("' ", ":");
    lon = lon.replace("\"", "");
    
    this.pos = DMS.DecodeLatLon(lat, lon);
  }
}

module.exports = Tower;

File: LoranCalc/LoranCalc/Tower.cs
using GeographicLib;

namespace LoranCalc;

public class Tower {
    public Position pos;
    public int codingDelay;
    

    public Tower(string lat, string lon, int codingDelay) {
        pos = new Position(lat, lon);
        this.codingDelay = codingDelay;
    }

    override public string ToString() {
        return $"{pos.lat}, {pos.lon}, {codingDelay}";
    }
}